config/300-apiserversource.yaml

```diff
@@ -118,6 +118,9 @@ spec:
                   controller:
                     type: boolean
                     description: "If true, emits the managing controller ref. Only supported for mode=Ref. More info: https://kubernetes.io/docs/concepts/workloads/controllers/garbage-collection/"
+                  labelSelector:
+                    type: object
+                    description: "Label selector restricting the list of objects to watch by their labels. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/"
```

Comment:
Do you want to mention the ability to OR label selectors with `|` or `;` (depending on the result of the above)?

Reply:
`;` does not appear at this level. Only the adapter can handle multiple label selectors.